examples/cassandra-3.11/start_cass_311.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+export CASSTAG=3.11.8
+export SGTAG=v1.0.7
+
+# Make sure backend-1, the seed node, is up before bringing up other nodes and stargate
+
+docker-compose up -d backend-1
+
+# Wait until the seed node is up before bringing up more nodes
+
+(docker-compose logs -f backend-1 &) | grep -q "Created default superuser role"
+
+# Bring up the 2nd C* node
+
+#docker-compose up -f $COMPOSE_FILE -d backend-2
+#(docker-compose logs -f backend-2 &) | grep -q "is now part of the cluster"
+
+# Bring up the 3rd C* node
+
+#docker-compose up -f $COMPOSE_FILE -d backend-3
+#(docker-compose logs -f backend-3 &) | grep -q "is now part of the cluster"
```

Comment:
Why aren't we bringing up the other two?

Reply:
removed comments